vipps-ecom-api.md

```diff
@@ -1156,22 +1156,18 @@ See [Timeouts](#timeouts) for details about timeouts.
 
 ## Userinfo
 
-**Important:** This is an early draft, this should be considered pilot
-functionality that we are currently rolling out in our test environemnt.
-Swagger will be fully updated
-shortly
-
-Vipps offers a functionality to ask for a generic consent to access Userinfo.
-This is based on the
-[Vipps Login](https://github.com/vippsas/vipps-login-api)
-solution, but merchants can seamlessly combine the two functionalites
-in a single user session. Combining both the userinfo and payment elements
+**Important:** This functionality is currently being rolled out in all environments.
+
+Vipps offers the possibility for merchants to ask for the users profile information as part of the payment flow. If the enduser has not already consented to sharing information from Vipps to the merchant the user will be asked for such consent before completing the payment flow. Once the payment flow is completed the merchant can get the profile information from our Userinfo endpoint. The Userinfo endpoint is shared with [Vipps login](https://github.com/vippsas/vipps-login-api) and the merchant needs to have activated Vipps login to use this feature. You find more information on how to activate Vipps login[here](https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api-faq.md#how-can-i-activate-and-set-up-vipps-login).
+A users consent to share information with a merchant applies accross our services. Thus, if the merchant implements Vipps login in addition to profile information as part of the payment flow, the merchant can also use Vipps to log the user in without the need for additional consents.
+
+Combining both the userinfo and payment elements
```

Comment:
This is covered by the information on the need to activate Vipps login. I thus think this can be deleted